doc/sphinx/source/vulkan.rst

```diff
@@ -0,0 +1,122 @@
+Vulkan driver
+=================
+
+NOTE: THIS DRIVER IS INCOMPLETE AND IS NOT ACTIVELY DEVELOPED BY TUNI.
+ANY MAINTENANCE WOULD NEED TO BE DONE BY COMMUNITY IN THEIR FREE TIME.
+
+PULL REQUESTS ON GITHUB WILL BE CONSIDERED, BUT REPORTS ON
+MISSING FEATURES WITHOUT CODE CONTRIBUTIONS ARE POINTLESS,
+BECAUSE LIMITATIONS ARE ALREADY KNOWN.
+
+This driver uses libvulkan and clspv to run OpenCL code on GPU devices via Vulkan API.
+
+Installation
+-------------
+
+Required:
+
+ * vulkan drivers (on Ubuntu, "mesa-vulkan-drivers" for opensource vulkan drivers)
+ * vulkan development files (on Ubuntu, "vulkan-headers" and "libvulkan-dev")
+ * SPIR-V tools (for clspv; on Ubuntu, package "spirv-tools")
+
+Optional:
+
+ * "vulkan-validationlayers-dev" for vulkan validation layers
+ * "vulkan-tools" package for vulkaninfo
```

Comment:
It's in "vulkan-utils" in Ubuntu 18.04 at least. Is this a wrong name or is it with this name in some distro?

Reply:
they renamed it somewhere between 18.04 and 20.04

```
a@Kaveri : ~ $ dpkg -s vulkan-tools
....
Replaces: vulkan-utils
```

added the older name to the docs.